src/main/java/frc/robot/constants/Constants.java

```diff
@@ -15,27 +15,31 @@ public final class Constants {
    * Operator Input Constants
    * @deprecated Instead of frc.robot.constants.Constants.oi use frc.robot.constants.OIConstants
    */
-  @Deprecated
-  public static final OIConstants oi = new OIConstants();
```

Comment:
You don't need to change this file

Reply:
Maybe, but I already did We aren't supposed to use Constants.oi anyway, so there is no reason to add it back.